docs/source/tutorials/introduction_to_python/basic_language_principles.rst

```diff
@@ -0,0 +1,70 @@
+.. _basic_language_principles:
+
+=========================
+Basic Language Principles 
+=========================
+
+-  Python is an interpreted language meaning there is no explicit
+   compilation step.
+
+-  The code is simply executed "as-is". This, coupled with the fact that
+   Python has a simple and easy-to-read syntax means it is an excellent
+   choice for a scripting language.
+
+-  It includes all of the features that one expects from a programming
+   language such as basic numerical types, a boolean, a string type and
+   support for various operations upon them. The table below summarises
+   how to use them in Python:
+
+| 
+
++---------+------------+------------------------------+
+| Type    | Example    | Python code                  |
++---------+------------+------------------------------+
+| integer | 5          | x = 5                        |
++---------+------------+------------------------------+
+| float   | 5.0        | x = 5.0                      |
++---------+------------+------------------------------+
+| boolean | True/False | x = True                     |
++---------+------------+------------------------------+
+| string  | 'python'   | x = 'python' or x = "python" |
++---------+------------+------------------------------+
+
+-  The operations are supported as long as it makes sense for that type,
+   e.g. there is no string division but ``+`` just means join the two
+   strings together.
+
+-  Variable assignment is simple than in other languages as you do not
+   have to declare the type and moreover it can be changed during
+   execution, e.g.
+
```

Comment:
Since the first following code example starts with import numpy, you may consider adding few infos about:
- module, packages
- famous python packages like numpy or SciPy with corresponding links